library/std/src/env.rs

```diff
@@ -490,8 +477,6 @@ pub struct JoinPathsError {
 ///     Ok(())
 /// }
 /// ```
```

Comment:
```suggestion
/// ```
///
/// [env::join_paths()]: join_paths
/// [env::split_paths()]: split_paths
```